src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -1,27 +1,30 @@
 package mate.academy.spring.controller;
 
+import javax.validation.Valid;
 import mate.academy.spring.dto.request.UserRequestDto;
 import mate.academy.spring.dto.response.UserResponseDto;
 import mate.academy.spring.model.User;
 import mate.academy.spring.service.AuthenticationService;
-import mate.academy.spring.service.mapper.UserMapper;
+import mate.academy.spring.service.mapper.ResponseDtoMapper;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class AuthenticationController {
     private final AuthenticationService authService;
-    private final UserMapper userMapper;
+    private final ResponseDtoMapper<UserResponseDto, User> userMapper;
 
-    public AuthenticationController(AuthenticationService authService, UserMapper userMapper) {
+    public AuthenticationController(AuthenticationService authService,
+                                    ResponseDtoMapper<UserResponseDto, User> userMapper) {
         this.authService = authService;
         this.userMapper = userMapper;
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
-        User user = authService.register(requestDto.getEmail(), requestDto.getPassword());
+    public UserResponseDto register(@RequestBody @Valid UserRequestDto requestDto) {
+        User user = authService.register(requestDto.getEmail(),
+                requestDto.getPassword(), requestDto.getRepeatPassword());
```

Comment:
you currently don't use repeatPassword inside register() method, so let's not pass it

Reply:
Yes...Ok